From 8d0bc79f38913b1a55e7d151b32bbc9462c24b47 Mon Sep 17 00:00:00 2001
From: Jason Miller <jason@jasom.org>
Date: Fri, 14 Aug 2015 19:03:09 -0700
Subject: [PATCH] Rename symbol to prevent conflict

One of the standard headers defines max_align_t on some versions of linux.

[Backported from upstream commit
https://github.com/mongrel2/mongrel2/commit/563bac8c59b9b32205164d237cf1ec0cb48d189f.]

Signed-off-by: Rodrigo Rebello <rprebello@gmail.com>
---
 src/mem/align.h  | 2 +-
 src/mem/halloc.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mem/align.h b/src/mem/align.h
index 4c6e183..03a4999 100644
--- a/src/mem/align.h
+++ b/src/mem/align.h
@@ -30,7 +30,7 @@ union max_align
 	void (*q)(void);
 };
 
-typedef union max_align max_align_t;
+typedef union max_align h_max_align_t;
 
 #endif
 
diff --git a/src/mem/halloc.c b/src/mem/halloc.c
index b097d1f..40d0c09 100644
--- a/src/mem/halloc.c
+++ b/src/mem/halloc.c
@@ -34,7 +34,7 @@ typedef struct hblock
 #endif
 	hlist_item_t  siblings; /* 2 pointers */
 	hlist_head_t  children; /* 1 pointer  */
-	max_align_t   data[1];  /* not allocated, see below */
+	h_max_align_t   data[1];  /* not allocated, see below */
 	
 } hblock_t;
 
-- 
2.1.4

